modules/standard/Reflection.chpl

```diff
@@ -132,13 +144,52 @@ proc getField(const ref x:?t, param s: string) type
    :returns: an rvalue referring to that field.
  */
 pragma "unsafe"
+inline
 proc getField(const ref x:?t, param s:string) const ref {
   param i = __primitive("field name to num", t, s);
   if i == 0 then
     compilerError("field ", s, " not found in ", t:string);
   return __primitive("field by num", x, i);
 }
 
+/* numImplementationFields() and getImplementationField()
+   allows querying non-record types that are implemented using records.
+   Restricting their applicability with isImplementedWithRecords()
+   prevents an internal error that currently would happen
+   when they are invoked on `int`, for example. Additional
+   types can be added to isImplementedWithRecords() as needed.
+*/
+
+pragma "no doc"
```

Comment:
> ... with regards to the public interface changes?

Based on your comment, I was under the impression that you were adding these to the public interface. However, they are no-doc'd. Is that intentional?